core/src/App.html

```diff
@@ -245,6 +248,10 @@
               '*'
             );
           }
+
+          if ('luigi.go-back-pressed' === e.data.msg) {
+            window.history.back();
```

Comment:
It works as it did before - you can go forward only once because Luigi stores only one step in history (I have no idea why). The only difference is now you have to click _forward_ only once to achieve the same effect. The history API won't let you detect if user pressed _back_ or _forward_ due to security reasons. It just emits `popstate` event in both cases. So I can say I've fixed it as far as I could. If it doesn't satisfy you, this should be done via another task in my opinion.

Reply:
You are right, don't know why it works like that..
So now it looks like it's fixed, but we should look into that in the future.